PYNCPPConfig.cmake.in

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 IHU Liryc, Université de Bordeaux, Inria.
+# License: BSD-3-Clause
+
+include(CMakeFindDependencyMacro)
+
+@PACKAGE_INIT@
+
+set_and_check(_project_binary_dir "@PACKAGE_PROJECT_BINARY_DIR@")
+
+file(GLOB _modules "${_project_binary_dir}/cmake/*.cmake")
+
+foreach(_module ${_modules})
+    include(${_module})
+endforeach()
+
+set_and_check(_pyncpp_export_file @PACKAGE_PYNCPP_EXPORT_FILE@)
+
+set(PYNCPP_Qt5_FOUND @PYNCPP_QT5_SUPPORT@)
+set(PYNCPP_SWIG_FOUND @PYNCPP_SWIG_SUPPORT@)
+
+set(PYNCPP_NO_GPL @PYNCPP_NO_GPL@)
+
+if(@PYNCPP_SWIG_SUPPORT@)
+    cmake_policy(SET CMP0078 NEW) # Standard SWIG target names
+    cmake_policy(SET CMP0086 NEW) # Pass -module option to SWIG
+    find_dependency(SWIG @PYNCPP_SWIG_VERSION@ REQUIRED COMPONENTS python)
+    include(${SWIG_USE_FILE})
+endif()
+
+if(@PYNCPP_CUSTOM_PYTHON@)
+    set(PYNCPP_LIBRARY_DIRS
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_1@"
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_2@"
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_3@"
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_4@"
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_5@"
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_6@"
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_7@"
+        "@PACKAGE_PYNCPP_LIBRARY_DIR_8@"
+        )
+    list(REMOVE_ITEM PYNCPP_LIBRARY_DIRS "")
```

Comment:
Yes. The number of directories is not fixed (may depend on platform and future components of pyncpp) and the way CMakePackageConfigHelpers is designed requires some hacking to adapt to that.

Reply:
(passing the list directly instead of using CMakePackageConfigHelpers's path variable mechanism would be simpler, but the resulting config file would not be relocatable)